debug/targets.py

```diff
@@ -79,6 +79,9 @@ class Target(object):
     # hardware will every do that.
     implements_custom_test = False
 
+    # Target uses the DTM Version 0.13 for memory accesses
+    uses_dtm_version_013 = False
```

Comment:
I don't think this is what you want. The debug spec doesn't say what should happen when reading invalid memory, and in any case the DTM is completely memory-agnostic. The DM (through abstract commands) deals with memory, but in our case it's mainly the hart implementation that determines the behavior. I would call this something like invalid_memory_returns_zero, with a comment saying that when true reading invalid memory doesn't return an error.

Reply:
Agree that this is more descriptive of what the flag actually means.